lib/calibrator.js

```diff
@@ -20,20 +20,16 @@ function Calibrator() {
  * @param {Browser} browser
  * @returns {Promise.<CalibrationResult>}
  */
-Calibrator.prototype.calibrate = function(browser) {
+Calibrator.prototype.calibrate = (browser) => {
```

Comment:
True, didn't pay attention to it. Maybe it's too early to apply such rules here, first prototypes has to be migrated to proper classes, to be able to use ES6 style